android/build.gradle

```diff
@@ -38,8 +38,8 @@ android {
 dependencies {
     //noinspection GradleDynamicVersion
     implementation "com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}"
-    implementation 'com.yoti.mobile.android.sdk:yoti-sdk-doc-scan:2.3.2'
-    implementation 'com.yoti.mobile.android.sdk:yoti-sdk-liveness-zoom:2.3.2'
+    implementation 'com.yoti.mobile.android.sdk:yoti-sdk-doc-scan:2.4.0'
+    implementation 'com.yoti.mobile.android.sdk:yoti-sdk-liveness-zoom:2.4.0'
```

Comment:
Oh! I forget to told you that we need to include another library for Supplementary document implementation 🙈 
Just add this lie below `liveness-zoom` one:
```
implementation 'com.yoti.mobile.android.sdk:yoti-sdk-doc-scan-sup:2.4.0'
```

Reply:
Added it in between just to keep the order of id doc, sup doc and liveness.